src/ProgressOnderwijsUtils/SchemaReflection/DatabaseDescription.cs

```diff
@@ -266,11 +290,13 @@ public sealed class View
     {
         readonly DbNamedObjectId view;
         public readonly DbColumnMetaData[] Columns;
+        public readonly Table[] ReferencedTables;
 
-        public View(DbNamedObjectId view, DbColumnMetaData[] columns)
+        public View(DbNamedObjectId view, DbColumnMetaData[] columns, Table[] referencedTables)
         {
             this.view = view;
             Columns = columns;
+            ReferencedTables = referencedTables;
         }
 
         public DbObjectId ObjectId
```

Comment:
Kun je deze ook weggooien, om fouten te voorkomen?

Reply:
laat maar, ik zie als dat dit niet triviaal is; out of scope.

Reply:
Bij de tabellen ook dan?

Reply:
in principe wel, maar ik zie dat we dat wel op diverse plekken gebruiken, dus ik vrees dat dit niet triviaal is, dus ... ooit, komt wel ;-)